fem/bilinearform_ext.cpp

```diff
@@ -94,9 +97,14 @@ void PABilinearFormExtension::Update()
    height = width = fes->GetVSize();
    trialFes = fes;
    testFes = fes;
-   ElementDofOrdering ordering = UsesTensorBasis(*a->FESpace())?
-                                 ElementDofOrdering::LEXICOGRAPHIC : ElementDofOrdering::NATIVE;
-   elem_restrict_lex = trialFes->GetElementRestriction(ordering);
+
+   const Operator* elem_restrict =
+      trialFes->GetElementRestriction(UsesTensorBasis(*a->FESpace())?
+                                      ElementDofOrdering::LEXICOGRAPHIC :
+                                      ElementDofOrdering::NATIVE);
+
+   elem_restrict_lex = dynamic_cast<const ElementRestriction*>(elem_restrict);
+
```

Comment:
And here

Reply:
@YohannDudouit, did you rename the above too?

Reply:
Yes, it should be done everywhere the name appears.